src/status_im2/common/bottom_sheet_screen/style.cljs

```diff
@@ -22,7 +22,8 @@
     :border-top-left-radius  20
     :border-top-right-radius 20
     :flex                    1
-    :overflow                :hidden}))
+    :overflow                :hidden
+    :padding-top             20}))
```

Comment:
Please remove this padding. It shouldn't be here. It doesn't look right on the photo-selector screen

Reply:
it should be here, because handle is absolute but we should reserve the place for it, so i'll fix photo-selector screen

Reply:
Hey @flexsurfer, it is not fixed. Check the scrolled state: https://www.figma.com/file/wA8Epdki2OWa8Vr067PCNQ/Composer-for-Mobile?type=design&node-id=6376-372393&t=2auDEJpU1f8tQVGS-0